avd_docs/aws/ec2/AVD-AWS-0164/Management_Console.md

```diff
@@ -0,0 +1,10 @@
+1. Sign into the management console
```

Comment:
This is a shell of the remediation - in the future it would have the screenshots as the CloudSploit remediation guides sometimes have